scripts/automation_scripts/update_lava_spec.py

```diff
@@ -0,0 +1,155 @@
+import argparse
+import json
+import os
+import re
+
+LAVA_PUBLIC_RPC = "public-rpc.lavanet.xyz:9090"
+
+
+def parse_endpoints_from_spec(lava_spec_file: str) -> dict[str, list[str]]:
+    print("### Parsing endpoints from current spec file")
+
+    endpoints: dict[str, list[str]] = {"grpc": [], "rest": []}
+
+    with open(lava_spec_file, "r", encoding="utf-8") as f:
+        content = json.load(f)
+
+    for api_collection in (
+        content.get("proposal", {}).get("specs", [])[0].get("api_collections", [])
+    ):
+        interface_type = api_collection.get("collection_data", {}).get(
+            "api_interface", ""
+        )
+
+        if interface_type not in endpoints:
+            continue
+
+        for api in api_collection.get("apis", []):
+            api_name = api.get("name", "")
+            if "lavanet" in api_name:
```

Comment:
instead of checking for lavanet (this only covers lava api's and is not generic)
lets use: not cosmwasm or not cosmos